source/elements/oneDAL/source/common_interface/error_handling.rst

```diff
@@ -1,3 +1,127 @@
+.. _error_handling:
+
 ==============
 Error handling
 ==============
+
+oneDAL error handling relies on mechanism of C++ exceptions. If an error occurs,
+it must be propagated at the point of a function call where it can be caught
+using С++ exception handlers.
+
+Exceptions classification
+=========================
+
+Classification is aligned with C++ STL classification. oneDAL base exception
```

Comment:
oneDAL shall introduce abstract classes defining the base for for hierarchy of exception classes.  Leaf classes in this hierarchy are derived/inherited from respective STL exception classes. oneDAL library shall throw only the exceptions represented with the leaf classes
- Do we have a better term than leaf?